google/__init__.py

```diff
@@ -35,15 +35,11 @@
 import sys
 import time
 
-if sys.version_info[0] > 2:
-    from http.cookiejar import LWPCookieJar
-    from urllib.request import Request, urlopen
-    from urllib.parse import quote_plus, urlparse, parse_qs
-else:
-    from cookielib import LWPCookieJar
-    from urllib import quote_plus
-    from urllib2 import Request, urlopen
-    from urlparse import urlparse, parse_qs
+
+from cookielib import LWPCookieJar
+from urllib import quote_plus
+from urllib2 import Request, urlopen
+from urlparse import urlparse, parse_qs
 
```

Comment:
Why are you removing support for Python 3? Did you mean to make these changes to this PR?